src/ui/GUIApplication.java

```diff
@@ -52,9 +52,15 @@ public void start(Stage stage) throws Exception {
 	
 	//High Scores screen
 	//@Override
-	public ObservableList<Score> start(Scene highScoresScene) throws Exception {
+	public void start(Scene highScoresScene) throws Exception {
 		Button back = new Button("back");
-		Stage window;
+		Stage window = null;
```

Comment:
In deze method moet geen window aangemaakt worden, die maken we al in de `start` method